app/lib/app/SecvisogramPage/View.js

```diff
@@ -794,9 +806,9 @@ function View({
                 formValues={formValues}
                 validationErrors={errors}
                 strict={strict}
+                sortButtonRef={sortButtonRef}
                 onSetStrict={onSetStrict}
                 onChange={onReplaceDoc}
-                onDownload={onDownload}
```

Comment:
Please remind me: Why did we remove this line?
And: The minimal-web-ui has a button `Export`, this branch has the `Download`one. `Export` is the correct one, right?